routes/index.js

```diff
@@ -64,7 +64,7 @@ module.exports = (keystone) => {
 				return next();
 			}
 
-			res.render('job', job);
+			res.redirect(job.url_base);
```

Comment:
Looking at this on my computer now.

Basically, I changed this to redirect to the url with the actual job listing if available, rather than trying to recreate the listing on our site. Given the variety of job listings I was seeing, this seemed simpler.

I did it as a redirect, so that the user still goes through a url on our site for each job. This gives us the option to not redirect and show the jobs on our site in the future.